fedmsg/meta/base.py

```diff
@@ -154,7 +154,9 @@ def handle_msg(self, msg, **config):
             return match.groups()[-1] or ""
 
     def title(self, msg, **config):
-        return '.'.join(msg['topic'].split('.')[3:])
+        if msg['topic'][0].isalpha():
```

Comment:
This checks that the first letter of the topic is alphabetic, right? What about checking that there are at least 3 `.` in there?

Also, thoughts on using ``msg['topic'].split('.', 3)[-1]`` below?